lib/common/websocket_subscriber.dart

```diff
@@ -0,0 +1,99 @@
+import 'common.dart';
+import 'common_desktop.dart';
+import 'package:collection/collection.dart';
+import 'package:lantern/plans/utils.dart';
+import 'package:fixnum/fixnum.dart';
+
+typedef void WebsocketChange();
+
+enum WebsocketMessage {
```

Comment:
Can change `WebsocketMessage` to `_WebsocketMessageType` also let's add _, I don't think we want to export the enum to anywhere except this class.

Reply:
Done